fenix/app/src/main/java/org/mozilla/fenix/translations/TranslationOptionsDialogBottomSheet.kt

```diff
@@ -52,53 +47,45 @@ fun TranslationOptionsDialogBottomSheet(
     onTranslationSettingsClicked: () -> Unit,
     aboutTranslationClicked: () -> Unit,
 ) {
-    Column(
-        modifier = Modifier
-            .background(
-                color = FirefoxTheme.colors.layer2,
-                shape = RoundedCornerShape(topStart = 8.dp, topEnd = 8.dp),
+    LazyColumn {
```

Comment:
Just double checking that I understand: Does this update make the first sheet scrollable via `LazyColumn`? 

Thanks for updating to the new switch too!

Reply:
Should the dialog header be scrollable? I would imagine the header shouldn't be though.

Reply:
> However, should the TranslationOptionsDialogHeader dialog header be scrollable? I would imagine the header shouldn't be though. 

I'm not entirely sure here, what is the usual pattern? I think justification for having the header be scrollable could be for accommodating long L10N strings and avoiding flexing too high. However, flex height only and not scrollable also makes sense for a header.

There was a note for [ensuring the lists themselves](https://www.figma.com/file/GT6oZLFw32LiCwKl5GOMWD?node-id=7212:177221&mode=design#573883735) were scrollable, but no mention on the header. A L10N request to ensure that [errors flexed](https://www.figma.com/file/GT6oZLFw32LiCwKl5GOMWD?node-id=7202:120020&mode=design#602754570) height also seems related to the discussion.

Reply:
Done , header outside list.